auditbeat/docs/modules/file_integrity.asciidoc

```diff
@@ -64,6 +68,12 @@ Linux.
 *`paths`*:: A list of paths (directories or files) to watch. Globs are
 not supported. The specified paths should exist when the metricset is started.
 
+*`exclude_files`:: A list of regular expressions used to filter out events
```

Comment:
Based on questions I've seen for Filebeat, I think it would be good to explicitly state that this regular expression is matched against the file's absolute path.